src/services/scanQueue.js

```diff
@@ -35,7 +40,8 @@ const processScanQueue = () => {
   scanQueue.on('failed', onFailedScanQueue);
   scanQueue.on('completed', onCompletedScanQueue);
   increaseListeners(scanQueue);
-  scanQueue.process((job) => processFile(job.data.key));
+  const process = (job) => processFile(job.data.key);
+  scanQueue.process(transactionQueueWrapper(process));
```

Comment:
Do we need to pass a second param here to the transactionQueueWrapper like in S3 and Resource queue's?

Reply:
the scan queue never passed a key, so there is no associated value to use as a context